### DISCUSSION TITLE
how config Version control management in server

### DISCUSSION THREAD

**Discussion by hktalent** [minio/minio#14121] [Q&A] [answered]
how config Version control management in server

Avoid malicious deletion by client users and bring trouble to management

In team management or company management

**Comment by klauspost** [comment, marked as answer]
See: 

https://docs.min.io/minio/baremetal/object-retention/minio-object-retention.html
https://docs.min.io/minio/baremetal/concepts/bucket-versioning.html
https://docs.min.io/minio/baremetal/object-retention/minio-object-locking.html

**Reply by hktalent** [reply]
@klauspost  Finally, I want to confirm that when uploading file objects, only the difference part is uploaded? Thank you very much

**Reply by hktalent** [reply]
@klauspost In addition, what is the maximum storage limit supported by the free version? Is the information currently available less than 1PB?

**Reply by klauspost** [reply]
> only the difference part is uploaded?

No. The S3 API only deals with whole objects.

**Reply by klauspost** [reply]
>  what is the maximum storage limit supported by the free version?

There is only one version. There are no limits.

**Reply by hktalent** [reply]
Thank you very much. I found that the version management function can only be enabled under the cluster, which is too complicated for a small team

**Reply by klauspost** [reply]
ok

**Comment by hktalent** [comment]
@klauspost thanks
